packages/frontend/src/components/Strategies/Crab/CrabPositionV2.tsx

```diff
@@ -82,14 +123,56 @@ const CrabPosition: React.FC = () => {
         </Box>
       ) : (
         <Box display="flex" alignItems="center" gridGap="20px" marginTop="16px" flexWrap="wrap">
-          <Metric
-            label="Position value"
-            value={
-              <Typography className={clsx(classes.metricValue, classes.white)}>
-                {formatCurrency(currentCrabPositionValue.toNumber())}
-              </Typography>
-            }
-          />
+          {usdcQueued.isGreaterThan(0) ? (
+            <Metric
+              label={
+                <MetricLabel
+                  label="Initiated Deposit"
+                  tooltipTitle="Your deposit has been initiated and will submitted via  auction to avoid price impact. This may take up until Tuesday"
```

Comment:
```suggestion
                  tooltipTitle="Your deposit has been initiated and will be submitted via auction to avoid price impact. This may take up until Tuesday"
```